spec/student_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../student'
+
+describe Student do
+  it 'Should create instance of the student class' do
+    student = Student.new(classroom: 6, age: 13, parent_permission: 'Yes', name: 'Mark')
+    expect(student).to be_an_instance_of Student
+  end
+end
+
+describe '#play_hooky' do
+  it 'should show play_hooky symbol' do
+    student = Student.new(classroom: 6, age: 13, parent_permission: 'Yes', name: 'Mark')
+    expect(student.play_hooky).to eq '¯\(ツ)/¯'
+  end
+end
```

Comment:
- [ ]  @team great job on your tests, however, you forgot to add tests for `classroom=` method, it's a project requirement to have unit tests for all methods in a class.